src/Microsoft.ML.StandardLearners/Standard/Simple/SimpleTrainers.cs

```diff
@@ -38,38 +39,37 @@ namespace Microsoft.ML.Runtime.Learners
     /// <summary>
     /// A trainer that trains a predictor that returns random values
     /// </summary>
-    public sealed class RandomTrainer : TrainerBase<RandomPredictor>
+
+    public sealed class RandomTrainer : TrainerEstimatorBase<BinaryPredictionTransformer<RandomPredictor>, RandomPredictor>
     {
         internal const string LoadNameValue = "RandomPredictor";
         internal const string UserNameValue = "Random Predictor";
         internal const string Summary = "A toy predictor that returns a random value.";
 
         public class Arguments
         {
-            // Some sample arguments
-            [Argument(ArgumentType.AtMostOnce, HelpText = "Learning rate", ShortName = "lr")]
-            public Float LearningRate = (Float)1.0;
-
-            [Argument(ArgumentType.AtMostOnce, HelpText = "Some bool arg", ShortName = "boolarg")]
-            public bool BooleanArg = false;
         }
 
         public override PredictionKind PredictionKind => PredictionKind.BinaryClassification;
 
         private static readonly TrainerInfo _info = new TrainerInfo(normalization: false, caching: false);
         public override TrainerInfo Info => _info;
 
-        public RandomTrainer(IHostEnvironment env, Arguments args)
```

Comment:
>RandomTrainer [](start = 15, length = 13)

You should not remove this constructor. Once you start writing the test you will discover that it is necessary.

Add a second constructor instead. Both here and for Prior